docs/DG_Diagrams/LogicComponent.puml

```diff
@@ -1,48 +1,57 @@
 @startuml
 
-allowmixing
+hide members
+hide circle
```

Comment:
Looks alot neater! But was just wondering what do ABC ,PQR and XYZ represent? And what about the commons component?

Reply:
The different command classes (ABC, XYZ, PQR) have been explained in the notes. As for commons component...I believe help and exit will be too trivial to mention

Reply:
Alright! This part looks good!

Reply:
Alright! This part looks good!